it/v4/endpoints/bfLoss/create/CreateBFLossControllerISpec.scala

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v4.endpoints.bfLoss.create
+
+import api.models.errors._
+import api.models.utils.JsonErrorValidators
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.HeaderNames.ACCEPT
+import play.api.http.Status._
+import play.api.libs.json._
+import play.api.libs.ws.{WSRequest, WSResponse}
+import play.api.test.Helpers.AUTHORIZATION
+import support.V4IntegrationBaseSpec
+import support.stubs.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
+
+class CreateBFLossControllerISpec extends V4IntegrationBaseSpec with JsonErrorValidators {
```

Comment:
Just noticed, the existing v5 ITs are also extending V4IntegrationBaseSpec. Could be worth updating them while you're there.

That said, these shouldn't really have "V4, V5 etc"-IntegrationBaseSpecs, just the one IntegrationBaseSpec, to be in line with the other ITs.   Up to you if you want to fix these now, or create a separate story to do so.